# Service that can be used to get orientation information about the EE using MoveIt.

---
float32 r
float32 p
float32 y
bool success
string message
